rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -251,6 +254,43 @@ def _create_networks(self, network_create_args):
         return lswitches
 
 
+    @atomic.action_timer("ovn_network.of_check_port")
+    def _of_check_ports(self, lports, sandboxes, port_bind_args):
```

Comment:
@huikang 
For question 1): Hopefully that simple change would work, but I haven't tried myself yet. I guess I won't have time this week, so it would be highly appreciated if you could try. Let me know if you'd like to take the ticket :)

For 2): I wonder if it is really needed to check correctness of flows in scale test. IMHO the focus here is the performance, and OVN functional tests should ensure correctness (the tests under OVS repo, and also openstack networking-ovn etc.). If we do suspect some scalability related bug, it may be helpful to have such kind of check AFTER the scale test complete. If we decide to implement such checks, we should make sure those checks won't impact the performance data. Instead of a separate thread running in parallel, I'd prefer a post-test validation. It should not be enabled by default because it would take too long considering the number of ports in scale test, and only enabled when we are testing correctness. Priority should not be high, though. What do you think?